crates/tools/src/lib.rs

```diff
@@ -1,4 +1,4 @@
-pub type Result<T> = std::result::Result<T, failure::Error>;
+use anyhow::Result;
```

Comment:
```suggestion
```

Reply:
You'll need to replace `Result<T>` with `Result<T, anyhow::Error>` too. This is best for consistency throughout the entire codebase.